WearTilesKotlin/app/src/main/java/com/example/wear/tiles/golden/Workout.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.wear.tiles.golden
+
+import android.content.Context
+import androidx.wear.tiles.ColorBuilders
+import androidx.wear.tiles.DeviceParametersBuilders.DeviceParameters
+import androidx.wear.tiles.ModifiersBuilders.Clickable
+import androidx.wear.tiles.material.Button
+import androidx.wear.tiles.material.ChipColors
+import androidx.wear.tiles.material.CompactChip
+import androidx.wear.tiles.material.Text
+import androidx.wear.tiles.material.TitleChip
+import androidx.wear.tiles.material.Typography
+import androidx.wear.tiles.material.layouts.MultiButtonLayout
+import androidx.wear.tiles.material.layouts.PrimaryLayout
+
+object Workout {
+    const val BUTTON_1_ICON_ID = "1"
+    const val BUTTON_2_ICON_ID = "2"
+    const val BUTTON_3_ICON_ID = "3"
+
+    fun buttonsLayout(
+        context: Context,
+        deviceParameters: DeviceParameters,
+        weekSummary: String,
+        button1Clickable: Clickable,
+        button2Clickable: Clickable,
+        button3Clickable: Clickable,
+        chipClickable: Clickable
+    ) =
+        PrimaryLayout.Builder(deviceParameters)
+            .setPrimaryLabelTextContent(
+                Text.Builder(context, weekSummary)
+                    .setTypography(Typography.TYPOGRAPHY_CAPTION1)
+                    .setColor(ColorBuilders.argb(GoldenTilesColors.Blue))
+                    .build()
+            )
+            .setContent(
+                MultiButtonLayout.Builder()
+                    .addButtonContent(
+                        Button.Builder(context, button1Clickable).setIconContent(BUTTON_1_ICON_ID)
+                            .build()
+                    )
+                    .addButtonContent(
+                        Button.Builder(context, button2Clickable).setIconContent(BUTTON_2_ICON_ID)
+                            .build()
+                    )
+                    .addButtonContent(
+                        Button.Builder(context, button3Clickable).setIconContent(BUTTON_3_ICON_ID)
+                            .build()
+                    )
+                    .build()
+            )
+            .setPrimaryChipContent(
+                CompactChip.Builder(context, "More", chipClickable, deviceParameters)
+                    .setChipColors(
+                        ChipColors(
```

Comment:
I know we've got variants of this where it selects primary/surface colors from a pseudo-theme object, but I haven't seen the utility yet.

Tiles are often (always?) on dark themes, and they're so small and each app would have so few of them, it seems easier to specify colors explicitly.

Reply:
https://github.com/yschimke/rememberwear/blob/main/wear/src/main/kotlin/com/google/wear/soyted/tile/RememberWearTileRenderer.kt#L116

Reply:
discussion merged, go here -> https://github.com/android/wear-os-samples/pull/450#discussion_r917425999